stream_test.go

```diff
@@ -27,6 +29,16 @@ var _ = Describe("Stream", func() {
 		mockFcm *mocks_fc.MockFlowControlManager
 	)
 
+	// in the tests for the stream deadlines we set a deadline
+	// and wait to make an assertion when Read / Write was unblocked
+	// on the CIs, the timing is a lot less precise, so scale every duration by this factor
+	getDeadlineScalingFactor := func() time.Duration {
+		if os.Getenv("CI") == "true" {
```

Comment:
I don't understand. At some point we will have to read the "magical" CI env var, if we want to detect that we're running on a CI.